samples/Samples.AspNetCoreMvc2/Samples.AspNetCoreMvc2.csproj

```diff
@@ -3,10 +3,7 @@
     <PropertyGroup>
         <TargetFrameworks>net461;netcoreapp2.0</TargetFrameworks>
         <MvcRazorCompileOnPublish>true</MvcRazorCompileOnPublish>
-    </PropertyGroup>
-
-    <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net461|AnyCPU'">
-      <PlatformTarget>x64</PlatformTarget>
```

Comment:
This was the line causing the `msbuild` error. Changing `PlatformTarget` to `x64` when `Platform` is `AnyCPU` breaks the build because the `project.assets.json` file won't have a target for `.NETFramework,Version=v4.6.1/win7-x86`.